### ISSUE TITLE
The development of RESTinio has been suspended

### ISSUE THREAD

**Issue by eao197** [Stiffstream/restinio#148]
RESTinio is being developed by our small company [stiffstream](https://stiffstream.com). Nobody pays us for that, we develop RESTinio using our own funds.

In the past, we got funding by doing custom development and/or maintenance of C/C++ projects by contracts. But the 2020 year was very unsuccessful for us and we didn't get the money we expected to spend for our OpenSource projects in 2021.

So now we have no resources to work on OpenSource projects including RESTinio and have to suspend the development of RESTinio for some time.

If you want to see some new functionality in RESTinio, then consider the possibility of ordering an extension of RESTinio on a paid basis.

You can also help us by spreading info about [stiffstream](https://stiffstream.com): maybe someone needs the help of experienced C++ developers. The RESTinio project is a good example of our abilities and the quality of our work.

I want to say "Thanks!" to all who selected RESTinio. Your attention to our project and your feedback was one of the main drivers in RESTinio's development.

**Comment by vidcentum** [comment]
@eao197 , I totally agree with you that open sources projects need to have either direct and indirect support. We will certainly talk about Stiffstream and explore any possibilities/collaboration of getting some paid assignments. 
Once again, our sincere thanks for building excellent libraries RESTinio and SObjectizer. Hope RESTinio will come back with C++17 implementation.
Best Regards
Maruthi

**Comment by eao197** [comment]
@vidcentum Thanks!

**Comment by giorgiozoppi** [comment]
@eao197  i am starting to look into it. Do you have time to help i need it?

**Comment by eao197** [comment]
@giorgiozoppi 
There is always a couple of hours per week to ask questions related to our open-source projects.

**Comment by giorgiozoppi** [comment]
i mean, i look into do some code, do you have a list of issues for ramping up?

**Comment by giorgiozoppi** [comment]
I would like to maintain this in the near future.

**Comment by eao197** [comment]
@giorgiozoppi 
I like to hear this. However, I see the current situation this way:

* if there will be some bug in RESTinio-0.6 we'll have time and resources to fix it;
* we haven't any plans to extend the functionality of RESTinio-0.6;
* our desires are centered on the next major branch 0.7 where we want to change the HTTP/1.1 parser (probably to our own one) and add a possibility to integrate support for HTTP/2 and HTTP/3 in the future. Because those are rather complex tasks and we didn't elaborate on them yet I don't think we can delegate to (or shared with) someone some part of that work.

**Comment by giorgiozoppi** [comment]
Ok. No problem, i will just play a bit with the branch 0.7.

**Comment by Majestro000** [comment]
Will it ever be resumed?

**Comment by eao197** [comment]
@Majestro000 
I hope so, but hardly this year.

Anyway, the project is not dead, and we're still here. We react to open issues. And if someone misses some functionality, then the development of such functionality can be ordered from [our company](https://stiffstream.com).
